sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/CaseImportExportSteps.java

```diff
@@ -102,20 +116,56 @@ public CaseImportExportSteps(
     When(
         "I check if downloaded data generated by custom case option is correct",
         () -> {
-          CustomCaseExportCSV reader =
-              parseCustomCaseExport(
-                  "./downloads/sormas_f\u00E4lle_" + LocalDate.now().format(formatter) + "_.csv");
+          String file =
+              "./downloads/sormas_f\u00E4lle_" + LocalDate.now().format(formatter) + "_.csv";
+          CustomCaseExportCSV reader = parseCustomCaseExport(file);
+          Path path = Paths.get(file);
+          Files.delete(path);
           softly.assertEquals(
               reader.getDisease(),
               DiseasesValues.getCaptionForName(apiState.getCreatedCase().getDisease()),
               "Diseases are not equal");
           softly.assertEquals(
-              reader.getFirstName(),
-              apiState.getLastCreatedPerson().getFirstName(),
+              String.format(reader.getFirstName(), Locale.GERMAN),
+              String.format(apiState.getLastCreatedPerson().getFirstName(), Locale.GERMAN),
               "First names are not equal");
           softly.assertEquals(
-              reader.getLastName(),
-              apiState.getLastCreatedPerson().getLastName(),
+              String.format(reader.getLastName(), Locale.GERMAN),
+              String.format(apiState.getLastCreatedPerson().getLastName(), Locale.GERMAN),
+              "Last names are not equal");
+          softly.assertAll();
+        });
+
+    When(
+        "I check if downloaded data generated by basic case option is correct",
+        () -> {
+          String file =
+              "./downloads/sormas_f\u00E4lle_" + LocalDate.now().format(formatter) + "_.csv";
```

Comment:
are we sure that the time difference is ok? after file is downloaded and we reach this line, at least 1ms will pass over for sure

Reply:
Yes, there is not time but date :)